docs/_data/toc.yml

```diff
@@ -78,19 +78,68 @@
     url: " https://support.shotgunsoftware.com/hc/en-us/articles/360024280254-Reference-Manual"
   - page: pipeline-integrations-reference
     children:
-    - text: core-api
-      url: " https://developer.shotgunsoftware.com/tk-core"
-    - text: qt-widgets
-      url: " https://developer.shotgunsoftware.com/tk-framework-qtwidgets/"
-    - text: shotgun-utils
-      url: " https://developer.shotgunsoftware.com/tk-framework-shotgunutils/"
-    - text: publish-api
-      url: " https://developer.shotgunsoftware.com/tk-multi-publish2/"
-    - text: hiero-export-api
-      url: " https://developer.shotgunsoftware.com/tk-hiero-export"
-    - text: file-system-configuration
-      url: " https://support.shotgunsoftware.com/hc/en-us/articles/219039868-Integrations-File-System-Reference"
-    - page: env-config-ref
+    - page: toolkit-apis
+      children:
+      - text: core-api
+        url: " https://developer.shotgunsoftware.com/tk-core"
+      - text: publish-api
+        url: " https://developer.shotgunsoftware.com/tk-multi-publish2/"
+      - text: hiero-export-api
+        url: " https://developer.shotgunsoftware.com/tk-hiero-export"
+    - page: toolkit-apps
+      children:
+      - page: tk-multi-about
+      - page: tk-multi-breakdown
+      - page: tk-multi-demo
+      - page: tk-multi-launchapp
+      - page: tk-multi-loader2
+      - page: tk-multi-publish2
+      - page: tk-multi-pythonconsole
+      - page: tk-multi-reviewsubmission
+      - page: tk-multi-screeningroom
+      - page: tk-multi-setframerange
+      - page: tk-multi-shotgunpanel
+      - page: tk-multi-snapshot
+      - page: tk-multi-workfiles2
+      - page: tk-hiero-export
+      - page: tk-hiero-openinshotgun
+      - page: tk-nuke-quickreview
+      - page: tk-nuke-writenode
+      - page: tk-shotgun-folders
+      - page: tk-shotgun-launchfolder
+      - page: tk-shotgun-launchpublish
+      - page: tk-flame-export
+      - page: tk-flame-review
+      - page: tk-houdini-alembicnode
+      - page: tk-houdini-mantranode
+      - page: tk-mari-projectmanager
+    - page: toolkit-engines
+      children:
+      - page: tk-3dsmax
+      - page: tk-aftereffects
+      - page: tk-alias
+      - page: tk-desktop
+      - page: tk-flame
+      - page: tk-houdini
+      - page: tk-mari
+      - page: tk-maya
+      - page: tk-motionbuilder
+      - page: tk-nuke
+      - page: tk-photoshopcc
+      - page: tk-shell
+      - page: tk-shotgun
+      - page: tk-vred
+    - page: toolkit-frameworks
+      children:
+      - text: qt-widgets
+        url: " https://developer.shotgunsoftware.com/tk-framework-qtwidgets/"
+      - text: shotgun-utils
+        url: " https://developer.shotgunsoftware.com/tk-framework-shotgunutils/"
+    - page: toolkit-configuration
```

Comment:
curious why we changed this one from Environment Config to Toolkit Config?

Reply:
I had an issue with the site build's page hashing routine that was giving me an error about duplicate page names. I suspect this was a caching issue with my local docker setup, but couldn't figure it out. The expedient way to handle it was to change the page name. Since it's not something a user sees it should be an invisible change.